### DISCUSSION TITLE
Total time driving and charging disappeared from trips dashboard

### DISCUSSION THREAD

**Discussion by joanquro** [teslamate-org/teslamate#2469] [Ideas]
Hello.

I'm starting this discussion because this is not really a bug, but it's a lost functionality IMHO.

I don't know since when this is happening, but I just noticed that in the trips dashboard the total time driving and charging no longer shows up under the pie chart. Now I can see the percentage of both things but I only get the detail when mouse hovering over them.

Is it possible to revert this change?. 

If possible, is it also possible adding the sum of both times just under them?. I would be very useful. 

Thank you.

<img width="441" alt="image" src="https://user-images.githubusercontent.com/62767148/153774749-03e31949-8d98-4f3a-b2f3-6b62fe176549.png">

**Comment by cwanja** [comment]
Looks like something changed maybe with the Grafana upgrade. Created a pull request for it. #2473

**Reply by joanquro** [reply]
Thanks a lot!
